core/report_model.py

```diff
@@ -174,7 +175,7 @@ class ReportForm(Form):
                     "List of the tests to include in the report. Only includes\
                  tasks that have ran in airflow."
                 ),
-                choices=get_all_test_choices(),
```

Comment:
Having removed this _here_ don't we want to set it in the views?

Reply:
`get_all_test_choices` was also being called in ReportModel's `create_form` and `edit_form`, which are the only times we need to generate a list of all the tests.

Reply:
So the functionality was already there, and we could just remove that line.